A set of patches to improve logs readability

alexey-tikhonov opened the pull request.

sumit-bose commented:
Hi,

thanks for the patches, they really help to make debug messages easier to understand.

You might want to fix the typo in the commit message of "krb5_chil: fixed few mistypes in debug messages" before the patch is committed.

bye,
Sumit

alexey-tikhonov commented:
> You might want to fix the typo in the commit message of "krb5_chil: fixed few mistypes in debug messages" before the patch is committed.

Thanks, fixed.

sumit-bose commented:
Thanks, ACK.

bye,
Sumit

pbrezina commented:
Pushed PR: https://github.com/SSSD/sssd/pull/5348

* `master`
    * 68497dc1a0aa26ce9ad1e39400020f961d937227 - krb5_child: adds verbosity
    * 445812769b65eba6c0c7b85a4ee5f653d8f4e700 - parse_krb5_child_response: adds verbosity
    * 038385dd9f41a82d7b74ecfc3257dce75dc54e46 - krb5_child: fixed few mistypes in debug messages
    * d1ed68bda6c1ba825ab903d7c6dc15e0e2774544 - UTILS: adds helper to convert authtok type to str
